test/manipulation.test.js

```diff
@@ -2615,25 +2615,25 @@ describe('manipulation', function() {
     it('preserves empty values from the database', async () => {
       // https://github.com/strongloop/loopback-datasource-juggler/issues/1692
 
-      // Initially, all Products were always active, no property was needed
-      const Product = db.define('Product', {name: String});
+      // Initially, all Players were always active, no property was needed
+      const Player = db.define('Player', {name: String});
 
-      await db.automigrate('Product');
-      const created = await Product.create({name: 'Pen'});
+      await db.automigrate('Player');
+      const created = await Player.create({name: 'Pen'});
 
       // Later on, we decide to introduce `active` property
-      Product.defineProperty('active', {
+      Player.defineProperty('active', {
         type: Boolean,
         default: false,
       });
+      await db.autoupdate('Player');
 
       // And upsertWithWhere an existing record
-      const found = await Product.upsertWithWhere({id: created.id}, {name: 'updated'});
-      found.toObject().should.eql({
-        id: created.id,
-        name: 'updated',
-        active: undefined,
-      });
+      const found = await Player.upsertWithWhere({id: created.id}, {name: 'updated'});
+      should(found.toObject().active).be.oneOf([
```

Comment:
Should we want to preserve the checks for `id` and `name` as well?

Reply:
Those properties are not really affected by default values, I feel it's ok to not check them - we have other tests in place to verify.